source/dem/input_parameter_inspection.cc

```diff
@@ -60,8 +60,10 @@ input_parameter_inspection(const DEMSolverParameters<dim> &dem_parameters,
     }
 
   // Insertion parameters checking
-  if (parameters.insertion_info.random_number_range >=
-      (0.5 * (parameters.insertion_info.distance_threshold - 1)))
+  if (parameters.insertion_info.insertion_method ==
+        Parameters::Lagrangian::InsertionInfo::InsertionMethod::non_uniform &&
+      parameters.insertion_info.random_number_range >=
+        (0.5 * (parameters.insertion_info.distance_threshold - 1)))
```

Comment:
Can you put the content of line 66 in a variable with a given name. Otherwise it's difficult to understand what it is supposed to mean.

Reply:
done.